Libraries/Image/ImageBackground.js

```diff
@@ -42,11 +42,23 @@ const View = require('View');
  * ```
  */
 class ImageBackground extends React.Component {
+  setNativeProps(props: Object) {
+    if (this._viewRef) {
+      this._viewRef.setNativeProps(props);
```

Comment:
Replaced it at https://github.com/facebook/react-native/pull/14884/commits/f94ff87698781194d90d939933028c23468b6444. Is that right?